pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -142,6 +142,7 @@ type osdProperties struct {
 	schedulerName       string
 	crushDeviceClass    string
 	encrypted           bool
+	name                string
```

Comment:
Let's be clear with the new properties that it's for the device set name. 
```suggestion
	deviceSetName                string
```

Reply:
Done.